### PR TITLE
Use Github API

### PR DESCRIPTION
**Pull request by kyoshidajp**
Thank you for a very nice tool 🥇

# Summary

Replace raw HTTP acces with Github API to access Github.

# Detail

To get user's public keys, call the following API via [go-github](https://github.com/google/go-github).
https://developer.github.com/v3/users/keys/#list-public-keys-for-a-user

I think that this way is useful considering the future extensibility.

### CODE REVIEW

File: pubkey.go [hunk 1]
```diff
@@ -79,18 +79,13 @@ func (p *PubKey) load() {
 
 // Fetch user's public key data from GitHub
 func (p *PubKey) FillKeys() *PubKey {
+	cli := github.NewClient(nil)
 	for u, _ := range p.users {
-		cli := &http.Client{Transport: p.transport}
```
Comment (potch8228) [inline pubkey.go:83]: I will be happy if HTTP transport setting is removed! 👍

**Review by potch8228 — CHANGES_REQUESTED** [review]

**Comment by potch8228** [comment]
BTW don't you need GitHub API Key if you use GitHub API??
Reason why I didn't choose such library is because of GitHub API Key.

File: pubkey.go (hunk 1 above)
Comment (kyoshidajp) [reply pubkey.go:83]: Thanks. I'm going to remove it tonight.

**Comment by kyoshidajp** [comment]
Actually, no need to authorized to call API. An unauthorized user can call API 60 requests per hour.  An authorized user who specifies access token can call it 5,000 requests per hour and access private repository.

For more information about RateLimit, see
https://developer.github.com/v3/#rate-limiting

**Review by potch8228 — APPROVED** [review]
Thanks!

**Pull request merged**